### ISSUE TITLE
[Mask RCNN/Tensorflow2] can not download resnet50 pretrained weight

### ISSUE THREAD

**Issue by ljh6389** [NVIDIA/DeepLearningExamples#598]
I tried to download resnet50 pretrained weight using '**download_and_process_pretrained_weights.sh**' , and got 404 not found error.
I think the bucket 'https://storage.googleapis.com/cloud-tpu-artifacts/resnet/resnet-nhwc-2018-10-14' doesn't exist anymore.

Can I download pretrained weights from other link?

**Comment by mmarcinkiewicz** [comment]
Hey, thanks for letting us know. I'm looking into this.

**Comment by pribalta** [comment]
@ljh6389 we merged the PR https://github.com/NVIDIA/DeepLearningExamples/pull/619 that should fix this
